src/test/java/com/hubspot/jinjava/lib/exptest/IsContainingExpTestTest.java

```diff
@@ -30,14 +30,14 @@ public void itPassesOnContainedValue() {
   }
 
   @Test
-  public void itFailsOnNullContainedValue() {
+  public void itPassesOnNullContainedValue() {
     assertThat(
         jinjava.render(
           String.format(CONTAINING_TEMPLATE, "[1, 2, null]", "null"),
           new HashMap<>()
         )
       )
-      .isEqualTo("fail");
+      .isEqualTo("pass");
```

Comment:
This is a breaking change. I don't see why this should fail before, and it does not fail in Jinja.